### ISSUE TITLE
[BUG] RuntimeWarning: A `OneCycleLR` scheduler is using 'interval': 'epoch'. Are you sure you didn't mean 'interval': 'step'?

### ISSUE THREAD

**Issue by fmerinocasallo** [unit8co/darts#2224]
I'm not 100 % comfortable tagging this issue as a bug but I couldn't find a more suitable label :grimacing: Please, re-tag it if you see fit :pray: 

**Describe the bug**
When using `optim.lr_scheduler.OneCycleLR` as my one cycle policy learning rate scheduler in combination with Darts' Torch Forecasting Models (e.g., `TFTModel`), I get the following warning:

*WARNING - (py.warnings._showwarnmsg) - ~/micromamba/envs/[...]/lib/python3.10/site-packages/pytorch_lightning/core/optimizer.py:299: RuntimeWarning: A `OneCycleLR` scheduler is using 'interval': 'epoch'. Are you sure you didn't mean 'interval': 'step'?*

**To Reproduce**
```
from darts.datasets import WeatherDataset
from darts.models import TFTModel
from torch.optim.lr_scheduler import OneCycleLR


series = WeatherDataset().load()
# predicting atmospheric pressure
target = series['p (mbar)'][:100]
# optionally, past observed rainfall (pretending to be unknown beyond index 100)
past_cov = series['rain (mm)'][:100]
# future temperatures (pretending this component is a forecast)
future_cov = series['T (degC)'][:106]

# by default, TFTModel is trained using a `QuantileRegression` making it a probabilistic forecasting model
max_lr = 0.9
n_epochs = 10
batch_size = 8
steps_per_epoch = (series.n_components + batch_size - 1)//batch_size
model = TFTModel(
    input_chunk_length=6,
    output_chunk_length=6,
    n_epochs=n_epochs,
    lr_scheduler_cls=OneCycleLR,
    lr_scheduler_kwargs={
        "max_lr": max_lr,
        "epochs": n_epochs,
        "steps_per_epoch": steps_per_epoch,
        # "total_steps": n_epochs*steps_per_epoch,
    },
)

# future_covariates are mandatory for `TFTModel`
model.fit(target, past_covariates=past_cov, future_covariates=future_cov)

# TFTModel is probabilistic by definition; using `num_samples >> 1` to generate probabilistic forecasts
pred = model.predict(6, num_samples=100)

```
**Expected behavior**
Darts should allow to configure the given optimizer through (PyTorch) Lightning's `lr_scheduler_config`.

**System (please complete the following information):**
 - Python version: 3.10.12
 - darts version 0.27.2
 - torch-lightning 1.9.4

**Additional context**
This issue is actually about pytorch-lightning. It has already been addressed on PyTorch's repo: https://github.com/pytorch/pytorch/issues/65417 and (Pytorch) Lightning's: https://github.com/Lightning-AI/pytorch-lightning/issues/9630

It seems just a matter of specifying `interval="step"` in the associated `lr_scheduler_config` when [configuring the given optimizer](https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.core.LightningModule.html#lightning.pytorch.core.LightningModule.configure_optimizers):

```
lr_scheduler_config = {
    # REQUIRED: The scheduler instance
    "scheduler": lr_scheduler,
    # The unit of the scheduler's step size, could also be 'step'.
    # 'epoch' updates the scheduler on epoch end whereas 'step'
    # updates it after a optimizer update.
    "interval": "step",
    # How many epochs/steps should pass between calls to
    # `scheduler.step()`. 1 corresponds to updating the learning
    # rate after every epoch/step.
    "frequency": 1,
    # Metric to to monitor for schedulers like `ReduceLROnPlateau`
    "monitor": "val_loss",
    # If set to `True`, will enforce that the value specified 'monitor'
    # is available when the scheduler is updated, thus stopping
    # training if not found. If set to `False`, it will only produce a warning
    "strict": True,
    # If using the `LearningRateMonitor` callback to monitor the
    # learning rate progress, this keyword can be used to specify
    # a custom logged name
    "name": None,
}
```

However, I cannot find an appropriate way of dealing with `lr_scheduler_config` through Darts.
